ibmcloud/terraform/cos/variables.tf

```diff
@@ -0,0 +1,21 @@
+##############################################################################
+# Input Variables
+##############################################################################
+
+# Resource Group Variables
+variable "resource_group_id" {
+  type        = string
+  description = "The resource group ID where the environment will be created"
+  default     = "default"
+}
+
+variable "ibmcloud_api_key" {
+  description = "API key to login to IBM Cloud"
+  type        = string
+}
+
+variable "ibm_region" {
+  description = "Name of the Region to deploy in to"
+  type        = string
+  default     = "jp-tok"
+}
```

Comment:
We can add variables for service instance name and bucket name. We need a default name for service instance, but we don't need a default name for bucket name, since it must be unique among all users.

Reply:
`bucket_name` and `cos_instance_name` variables are now in `variables.tf`. 
`cos_instance_name` now has a default of `cos-image-instance`.
`bucket_name` does not have a default value.